Cargo.toml

```diff
@@ -58,3 +58,7 @@ criterion = { version = "0.4.0", features = ["async_futures"] }
 [[bench]]
 name = "bench"
 harness = false
+
+[[bench]]
+name = "alg-bench"
```

Comment:
maybe call it reachability-bench or smth

Reply:
Fixed